frontend/src/ListView.js

```diff
@@ -1,4 +1,10 @@
 import React, { Component } from "react";
+import Home from "./Home";
+import ReactDOM from "react-dom";
+
+function goHome(){//add browser router??
```

Comment:
remove comment. will make note of this in github

Reply:
good idea